bin/bundle

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env ruby2.1
```

Comment:
We can use the ruby directive in the Gemfile to do this. http://bundler.io/v1.3/gemfile_ruby.html